Microsoft.Toolkit.Uwp.UI.Animations/Builders/AnimationBuilder.cs

```diff
@@ -48,9 +48,9 @@ private AnimationBuilder()
         /// </para>
         /// For instance:
         /// <code>
-        /// var animation = AnimationBuilder.Create().Opacity(0, 1).Size(1.2, 1);<br/>
-        /// <br/>
-        /// animation.Start(MyButton);<br/>
+        /// var animation = AnimationBuilder.Create().Opacity(0, 1).Size(1.2, 1);
+        /// 
```

Comment:
```suggestion
        ///
```
D:\a\1\s\Microsoft.Toolkit.Uwp.UI.Animations\Builders\AnimationBuilder.cs(52,12): error SA1028: Code should not contain trailing whitespace [D:\a\1\s\Microsoft.Toolkit.Uwp.UI.Animations\Microsoft.Toolkit.Uwp.UI.Animations.csproj]